daq_macros.py

```diff
@@ -232,6 +232,42 @@ def changeImageCenterHighMag(x,y,czoom):
 def run_top_view_optimized():
     RE(topview_optimized())
 
+def run_on_mount_option(sample_id):
```

Comment:
This function is used to provide interactive users the ability to 
1. Do nothing on mount 
2. Only center sample
3. Center sample and take 2 orthogonal rasters of the sample